src/domain/car.js

```diff
@@ -0,0 +1,32 @@
+import { ERROR_NAME_LABEL } from "../util/validation.js";
+
+const VALID_NAME_LENGTH = 5;
+const DEFAULT_CAR_NOW = 0;
+
+/**
+ * @type {import("./race").Target}
+ */
+
+export function Car(name, now = DEFAULT_CAR_NOW) {
+  if (!isVaildName(name)) {
+    throw new Error(ERROR_NAME_LABEL);
+  }
+
+  function setMovement(distance) {
+    now += distance;
+  }
+
+  function getNow() {
+    return now;
+  }
+
+  function getName() {
+    return name;
+  }
+
+  return { setMovement, getNow, getName };
```

Comment:
```suggestion
  function getNow() {
    return now;
  }

  function getName() {
    return name;
  }

  return { setMovement, get now(){ return now }, get name() { return name } };
```
이렇게 getter로 표현하는 방법도 있답니다!

Reply:
오오 한 번 적용해보면서 빠르게 문서 읽어봤습니다..! 👍 

[프로퍼티 getter와 setter](https://ko.javascript.info/property-accessors#ref-885)

910db530c48e105a7bf8419ff65b18bfa45fdad9